.github/workflows/php-compatibility.yml

```diff
@@ -0,0 +1,18 @@
+name: PHP Compatibility
+
+on:
+  pull_request
```

Comment:
Do we want to specify target branches for this?

Reply:
I don't think so. The point is to execute the script on every pull request to make authors aware of risky PHP code, and this line will run it on the PR branch. Do you think we need to?

Reply:
My thought was to not keep the action workers busy on temporary branches we might push but since the workflow doesn't take that long I think it is okay to run it on every branch.